x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/action/DeleteExpiredDataAction.java

```diff
@@ -31,20 +34,97 @@ private DeleteExpiredDataAction() {
 
     public static class Request extends ActionRequest {
 
+        public static final ParseField REQUESTS_PER_SECOND = new ParseField("requests_per_second");
+        public static final ParseField TIMEOUT = new ParseField("timeout");
+
+        public static final ObjectParser<Request, Void> PARSER = new ObjectParser<>(
+            "delete_expired_data_request",
+            true,
```

Comment:
We would usually error on unknown fields when parsing REST requests.  Should this be `false` on the server side?

Reply:
yes yes, it should be false. Fixing!